docs/api.md

```diff
@@ -18,25 +18,25 @@ You can run Yargs without any configuration, and it will do its
 best to parse `process.argv`:
 
 ```javascript
-require('yargs').argv
+require('yargs/yargs')(process.argv.slice(2)).argv
```

Comment:
The introduction to this first example as well as the following example should be changed. Currently, the introductions to these code examples say they show calling `yargs` without passing arguments and then with passing arguments.  However, with `yargs/yargs`, arguments must always be passed, so there’s no such distinction.

First drafts of more appropriate introductions for examples 1 and 2:

> You can run Yargs directly on the relevant part of `process.argv`:

> Yargs accepts any array of options. It doesn’t have to be from `process.argv`:

The change in this PR is nonetheless better than leaving the docs as recommending `require('yargs')`, so I would be fine with merging this PR before this problem is fixed and then fixing it later.